### ISSUE TITLE
Can I get metrics like CPU usage ,network traffic,the state of container:running or stopped?

### ISSUE THREAD

**Issue by isabelnoronha61** [prometheus-net/docker_exporter#9]
I am runing a docker service which has nearly 2K containers for simulation purpose and I would like to monitor if the container stops and restarts inside the service or send an alert if container has stopped..

**Comment by sandersaares** [comment]
Yes, the exporter exposes all the metrics you listed in your question.

However, the exporter is not really optimized for situations involving 2000 containers. The Docker APIs it uses are quite low-performance and exhibit lots of latency and high "loading times". I estimate your experience with 2000 container scenarios would not be satisfactory.

In theory, there are some adjustments that can be made to make the exporter more suitable for this scenario but at the moment they are not on the roadmap (though if sufficiently important to you, perhaps sponsoring feature development on this could be considered).
